apps/services/routers.py

```diff
@@ -139,7 +139,7 @@ async def read_user_request_booking_hostel(
 
 @services_router.put("/{university_id}/user-request/{user_request_id}",
                      name="update_cancel_user_request",
-                     response_model=JSENDOutSchema[CancelRequestOut],
+                     response_model=JSENDOutSchema[CreateUserRequestOut],
```

Comment:
`CreateUserRequestOut` in `update_cancel_user_request`
maybe should rename or make inheritance in `CreateUserRequestOut` from same `CancelRequestOut`

Reply:
Really don't know, may be. But it will only lead to grow code amount. Let Konstantin and Anton will their advise.